parsec/backend/organization.py

```diff
@@ -328,6 +340,8 @@ async def api_organization_bootstrap(self, client_ctx, msg):
         except (OrganizationNotFoundError, OrganizationInvalidBootstrapTokenError):
             return {"status": "not_found"}
 
+        organization = await self.get(client_ctx.organization_id)
```

Comment:
Same as for `organization_create`: the client is used to call `organization_stats` whenever `users_limit` info is needed

On top of that it is a bit dangerous to do this call here because we can endup in a situation where `self.bootstrap` succeeded, but `self.get` failed (e.g. PostgreSQL DB entered in maintenance in the meantime), then returning the wrong status to the client

Not we shouldn't have this trouble with `await self.webhooks.on_organization_bootstrap` given in case of error, this function is designed to log the error and return without exceptions.

...

Now that I've written this explanation, I'm kind of sad it is not in the codebase as a comment... maybe you could add it for me ? 😄

Reply:
ping !